docs/plugins/outputs/librato.asciidoc

```diff
@@ -154,24 +143,19 @@ Send data to Librato as a gauge
 
 Example:
 [source,ruby]
------
     {
         "value" => "%{bytes_received}"
         "source" => "%{host}"
         "name" => "apache_bytes"
     }
------
 Additionally, you can override the `measure_time` for the event. Must be a unix timestamp:
-
 [source,ruby]
------
     {
         "value" => "%{bytes_received}"
         "source" => "%{host}"
         "name" => "apache_bytes"
         "measure_time" => "%{my_unixtime_field}
     }
------
 Default is to use the event's timestamp
 
```

Comment:
REGRESSION:  Additional formatting required for proper rendering with --asciidoctor.